managed/services/management/node_test.go

```diff
@@ -38,7 +36,15 @@ import (
 )
 
 func TestNodeService(t *testing.T) {
-	t.Run("Register", func(t *testing.T) {
+	var authProvider mockAuthProvider
+	authProvider.Test(t)
+
+	serviceAccountID := int(0)
+	nodeName := "test-node"
+	reregister := false
+	force := true
```

Comment:
I don't like an idea of using global variables, it won't allow us to run tests in parallel.
As far as I see we can replace them in each place just by actual value in each place or use local variable.
And all of them are used within `Register/Unregister -> New` test group, so it's better to put them there